modules/Bio/EnsEMBL/Variation/DBSQL/vcf_config.json

```diff
@@ -348,6 +357,34 @@
         }
       }
     },
+    { "id": "esp_GRCh37",
+      "species": "homo_sapiens",
+      "assembly": "GRCh37",
+      "type": "remote",
+      "filename_template": "ftp://ftp.ensembl.org/pub/data_files/homo_sapiens/GRCh37/variation_genotype/ESP6500SI-V2-SSA137.vcf.gz",
```

Comment:
Is this file only made available at time of release?

Reply:
yes